src/Components/Web.JS/src/GlobalExports.ts

```diff
@@ -48,7 +48,10 @@ interface IBlazor {
     getLazyAssemblies?: any
     dotNetCriticalError?: any
     getSatelliteAssemblies?: any,
-    applyHotReload?: (id: string, metadataDelta: string, ilDelta: string) => void
+    HotReload: {
```

Comment:
`hotReload`?

Reply:
It's probably following the convention that `Virtualize`/`InputFile` uses which is pascal case for things that play a namespace-like role.

TBH all the `Blazor._internal.*` area is quite ad-hoc and less consistent than most of our other naming. We've chosen not to prioritize it so far because it's not public API.

Reply:
Ok, I made a quick update and I'm planning to leave this in the top-level. It makes the change compatible with dotnet-watch regardless of if we are able to make the corresponding change there.